spec/02-integration/02-cmd/02-start_stop_spec.lua

```diff
@@ -182,7 +182,8 @@ describe("kong start/stop", function()
         thread:join()
       end)
 
-      local ok, err = helpers.kong_exec("start --conf " .. helpers.test_conf_path)
+      local ok, err = helpers.start_kong({ ngx_conf = helpers.test_conf_path })
```

Comment:
I am slightly confused, because `ngx_conf` is not an recognized value for this table of options given to `start_kong`. The option is named `nginx_kong` instead.

Additionally, `test_conf_path` will be a Kong configuration file, and not an Nginx template, so even if the key was to be renamed to the `nginx_conf` option, this is likely to fail as well.

I also seems to me like the culprit for a test that deletes the system prefix is more likely to be related to a wrong `prefix` argument?